pkg/ddc/alluxio/operations/cached.go

```diff
@@ -61,17 +61,32 @@ func (a AlluxioFileUtils) CachedState() (cached int64, err error) {
 // clean cache with a preset timeout of 60s
 func (a AlluxioFileUtils) CleanCache(path string) (err error) {
 	var (
-		// todo(TrafalgarZZZ): adaptively exec timeout command with proper option according to Alluxio image version
-		command = []string{"timeout", "60", "alluxio", "fs", "free", "-f", path}
-		stdout  string
-		stderr  string
+		releaseVersion   = []string{"cat", "/etc/issue"}
+		commandForUbuntu = []string{"timeout", "60", "alluxio", "fs", "free", "-f", path}
+		commandForAlpine = []string{"timeout", "-t", "60", "alluxio", "fs", "free", "-f", path}
+		command          []string
+		stdout           string
+		stderr           string
 	)
 
+	stdout, stderr, err = a.exec(releaseVersion, false)
+	if err != nil {
+		err = fmt.Errorf("execute command %v with expectedErr: %v stdout %s and stderr %s", releaseVersion, err, stdout, stderr)
+	}
+
+	if strings.Contains(stdout, "Ubuntu") {
```

Comment:
Will the output be `ubuntu` ?

Reply:
I also suggest to test, and show sample result in the comments. Thanks very much.

Reply:
I have already tested the code yesterday. I checked the code with the alluxio images (release-2.5.0-1,release-2.5.0-2) separately.  The logs of the alluxioruntime controller showed that different commands executed. The logs are like this.

``
 2021-05-22T22:10:28.295+0800    INFO    alluxioctl.AlluxioRuntime       operations/base.go:450  execute in time {"alluxioruntime": "default/hbase", "command": ["timeout", "60", "alluxio", "fs", "free", "-f", "/"]}
``
Thanks for the suggestions.